ui/packages/consul-ui/app/abilities/intention.js

```diff
@@ -4,7 +4,12 @@ export default class IntentionAbility extends BaseAbility {
   resource = 'intention';
 
   get canWrite() {
-    return super.canWrite && (typeof this.item === 'undefined' || !this.canViewCRD);
+    // Peered intentions aren't writable
+    if(typeof this.item !== 'undefined' && typeof this.item.SourcePeer !== 'undefined') {
```

Comment:
The changes to the the intentions model below renames `SourcePeer` to `PeerName`; is this logic still accurate? Or does the reference from `this.item.SourcePeer` need to change to `this.item.PeerName`?

If `SourcePeer` is still correct, where does that come from?